src/packages.builds

```diff
@@ -3,7 +3,6 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <AdditionalProperties>BuildPackageLibraryReferences=false</AdditionalProperties>
```

Comment:
@ericstj I assume we need this because we are doing project references to pkgproj's now but do we know what impact this might have on our overall build?